tools/ClassGen/NodeGen.cpp

```diff
@@ -318,6 +318,22 @@ int main(int argc, char **argv) {
                     "input vector of length N the output is a Nx2 matrix with "
                     "(offset, lengths) packaged for each segment.");
 
+  // clang-format off
+  BB.newNode("SparseToDense")
+      .addInput("Indices")
+      .addInput("Values")
+      .addResultFromCtorArg()
+      .setDocstring(
+          "Converts the sparse representation specified by the pair "
+          "(Indices, Values) into a dense one. This dense "
+          "representation is as large as the maximum index in Indices "
```

Comment:
`As large as the maximum index` <- this is not true. We use 3rd input to figure out output size.